client/view.js

```diff
@@ -85,7 +91,9 @@ export default class View
 	#handsUpdated(hands)
 	{
 		for(var h = 0; h < hands.length; h++)
-			this.#updatePlayer(hands[h]);
+			{

```

Comment:
current code formatting guidelines (lets make VSCode enforce these) doesnt use braces for a single-lined blocked. And even if we did, it doesn't need double indenting ;)